packages/server/src/auth/newuser.ts

```diff
@@ -31,12 +32,6 @@ export async function newUserHandler(req: Request, res: Response): Promise<void>
     return;
   }
 
-  const errors = validationResult(req);
-  if (!errors.isEmpty()) {
-    sendOutcome(res, invalidRequest(errors));
-    return;
-  }
-
```

Comment:
Not sure if this approach is viable given that the behavior is slightly different here in that request validation would now run prior to the config check

Reply:
It's ok 👍